charts/app-config/values.yaml

```diff
@@ -198,3 +198,22 @@ jdbc:
   password: password
   # -- JDBC dialect to use for JDBC Connection
   dialect: org.hibernate.dialect.PostgreSQLDialect
+
+# Sentry monitoring configuration
+sentry:
+  # -- DSN (Data Source Name) of the sentry server
+  dsn:
+  # -- Log level for sentry (TRACE, DEBUG, INFO, WARN, or ERROR)
+  level: ERROR
+  service:
+    # -- Name of the service to be displayed in the sentry dashboard
+    name:
+    # -- App version to be displayed in the sentry dashboard
+    release:
+    # -- Environment of the sentry service
+    environment: production
+  stacktrace:
+    # -- Set to true, if stack trace should be enabled
+    enabled: true
+    # -- Comma-separated list of package prefixes to be included in the stacktrace
+    packages: "org.radarbase.appconfig,org.radarbase.lang,comparisons"
```

Comment:
Is this package string correct? The `comparisons` part looks strange to me.

Reply:
this is the one that is packaged under comparisons:
[radar-expression-lang/src/main/kotlin/comparisons/Library.kt](https://github.com/RADAR-base/radar-app-config/blob/dev/radar-expression-lang/src/main/kotlin/comparisons/Library.kt)
If not needed, I can remove it

Reply:
@ewelinagr Did you perhaps miss this comment?